Add Chopin Mandarin tutorial and update beta user info (#46)
apiobuild/apiobuild.github.io: content/blog/how-to-create-web-store-with-apio.md

- name: unique name/id contains **only dashes, underscores, and alphanumeric are allowed** for each product
- nickname: item names that will appear on the store page, they can be **any language, symbol, and even emoji**
- description *(optional)*: more information about the product
- product_url *(optional)*: link to an external web page
- image_url: url of the product image
- price: product price
- max_qty: maximum quantity that one customer can buy. **If max_qty = 0, it will show 'Out of Stock'**.

> - category *(optional)*: to allow customer to filter products. Multiple categories are allowed

Reviewer: ```suggestion
- category *(optional)*: to allow customer to filter products. Multiple categories can be separated by comma.
```
